Comment Section

realyuvishere opened the pull request:
### What is the change?
Comment section module is added to the products

### What does it fix/add?
It allows users to comment on a particular product

### How was it tested?
```console.log()``` and using the command ```npm run dev```

## Submissions guide:
- [yes] Have you followed the [Contribution guide](https://github.com/Neilblaze/Anyxz/blob/master/CONTRIBUTING.md)?
- [yes] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Neilblaze/Anyxz/pulls) for the same update/change?
- [yes] Have you made corresponding changes to the documentation?
- [yes] Your submission doesn't break any existing feature.
- [yes] Have you lint your code locally prior to submission?

### Screenshots (if appropriate):
![comment section](https://user-images.githubusercontent.com/21034233/109388395-beb53280-792c-11eb-94e5-cb888f1767b1.gif)

Commit:
Merge remote-tracking branch 'ALPHAVIO/main' into main

Commit:
Merge remote-tracking branch 'ALPHAVIO/main' into main

Commit:
Merge remote-tracking branch 'ALPHAVIO/main' into main

Commit:
Comment section - initial commit

> Main models created for comments
> Created routes and linked to the index.js file
> Created ejs view for comment section
> Added font awesome to the views

Commit:
Comment section implemented

> The comments are saved to the particular product
> The comments DOES NOT save the author of the comment or the product
yet
> The comments are rendered when routed to particular path containing
the _id of the product.

Pranav016 reviewed (changes requested):
* When we click on a product, it should show the product name with its description along with all its comments below it. Right now its showing only the comments. Just add a new parameter of the product name and desc. to the ejs template for this route.
![image](https://user-images.githubusercontent.com/54665036/109433363-f3f97780-7a35-11eb-8751-7514a1e0357b.png)

* Make the product an extended link using extended link property of cards. (optional right now, can be implemented in next few pull requests)
* When an unauthenticated user is trying to comment on a product just redirect to login page.

Commit:
Product details on comment

> The details of the specific product are displayed along with the
comments
> The product card in the marketplace is a stretched link

Pranav016 reviewed (approved):
Great work.
